releases/v0.15.0-rc0.yaml

```diff
@@ -3,7 +3,10 @@ version: v0.15.0-rc0
 name: 0.15.0-rc0
 pre-release: true
 branch: release-0.15
-status: admiral
+status: projects
 components:
   shipyard: 9be8e3ff84725d3af9c7cc215d6bdfa8990723bb
   admiral: 46cb4212e940082bedcfdf551b8324a3807cb84f
+  cloud-prepare: e6113da1f6832c4ef0dc15885557dccae8f2be7c
+  lighthouse: 873b36a324c9cbc1abe473788dce95e408e026a0
+  submariner: 9f03a0544283ce536836595bbae38c94f95de1f9
```

Comment:
This is going to need to be updated to https://github.com/submariner-io/submariner/pull/2374/commits/1faf6115599abbc5d03cbcd92872a5352826a60c once that merges. Perhaps just close this PR and re-run `make`.

Reply:
Alternatively, try `make release UPDATE=yes` ;)